tests/testapp/models.py

```diff
@@ -47,10 +47,18 @@ class Cover(models.Model):
 
 class Page(models.Model):
     title = models.CharField(max_length=100)
+    alt_page_id = models.CharField(max_length=100, unique=True)
     content = models.TextField()
     previous_page = models.ForeignKey('self', null=True, on_delete=models.CASCADE)
 
 
+class PageNote(models.Model):
+    page = models.ForeignKey(Page, to_field='alt_page_id', on_delete=models.CASCADE)
```

Comment:
I had trouble figuring out a good example with the test data models. This PageNote would be like a note on a specific page, like an annotation, i suppose. But, it references the `alt_page_id` of the `Page`

Reply:
What I'm probably going to do is just define a new set of models specifically for testing this.